packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -135,18 +135,20 @@ export const Table = styled.table<{ $showFooter?: boolean }>`
     }
 
     .sticky-column {
-        position: sticky !important;
```

Comment:
this was an unnecessary rule and actually caused styling issues as you were scrolling the table.